assets/xml/audio/samplebanks/SampleBank_0.xml

```diff
@@ -438,21 +438,21 @@
     <Sample Name="SAMPLE_0_435" FileName="Sample435" Offset="0x2F1780" SampleRate="16000" BaseNote="C4"/>
     <Sample Name="SAMPLE_0_436" FileName="Sample436" Offset="0x2F3850" SampleRate="16000" BaseNote="C4"/>
     <Sample Name="SAMPLE_0_437" FileName="Sample437" Offset="0x2F5360" SampleRate="16000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_438" FileName="Sample438" Offset="0x2F63A0" SampleRate="16000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_439" FileName="Sample439" Offset="0x2F68B0" SampleRate="16000" BaseNote="C0"/>
-    <Sample Name="SAMPLE_0_440" FileName="Sample440" Offset="0x2F7490" SampleRate="16000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_441" FileName="Sample441" Offset="0x2F87D0" SampleRate="16000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_442" FileName="Sample442" Offset="0x2FC270" SampleRate="16000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_443" FileName="Sample443" Offset="0x2FFF70" SampleRate="16000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_444" FileName="Sample444" Offset="0x3030E0" SampleRate="16000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_445" FileName="Sample445" Offset="0x303670" SampleRate="16000" BaseNote="C0"/>
-    <Sample Name="SAMPLE_0_446" FileName="Sample446" Offset="0x304E90" SampleRate="16000" BaseNote="C0"/>
+    <Sample Name="DekuBabaChomp" FileName="deku_baba_chomp" Offset="0x2F63A0" SampleRate="16000" BaseNote="C4"/>
+    <Sample Name="DekuBabaLunge" FileName="deku_baba_lunge" Offset="0x2F68B0" SampleRate="16000" BaseNote="C0"/>
+    <Sample Name="Buzzing" FileName="buzzing" Offset="0x2F7490" SampleRate="16000" BaseNote="C4"/>
+    <Sample Name="ElectricBurst" FileName="electric_burst" Offset="0x2F87D0" SampleRate="16000" BaseNote="C4"/>
+    <Sample Name="PhantomGanoninAgon" FileName="phantom_ganonin_agon" Offset="0x2FC270" SampleRate="16000" BaseNote="C4"/>
```

Comment:
what does "ganonin agon" mean?

Reply:
Looks like a copy paste error. Should have been `PhantomGanonInAgony`